cli/Squidex.CLI/Squidex.CLI/Commands/App_Sync.cs

```diff
@@ -183,6 +186,11 @@ public Validator()
                     RuleFor(x => x.Folder).NotEmpty();
                 }
             }
+
+            private DateTimeOffset GetMaxAgeDate()
+            {
+                return MaxAge == null ? DateTimeOffset.MinValue : new DateTimeOffset(DateTime.Today.AddDays(-(int)MaxAge));
```

Comment:
Is timespan not supported directly? Or another unit like hours? I think it would be useful to sync everything since a short timeframe.